nuget/Microsoft.Windows.CppWinRT.nuspec

```diff
@@ -13,7 +13,6 @@
     <copyright>© Microsoft Corporation. All rights reserved.</copyright>
     <license type="file">LICENSE</license>
     <projectUrl>https://github.com/Microsoft/xlang/tree/master/src/package/cppwinrt/nuget</projectUrl>
-    <iconUrl>https://aka.ms/cppwinrt.ico</iconUrl>
```

Comment:
why was this removed?

Reply:
The `iconUrl` tag is [deprecated](https://docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5048), and produces a warning during `nuget pack`. I attempted to download this URL and include the icon the the package using the non-deprecated `icon` tag, but the URL given 404ed when I tried that.

Reply:
FYI - I've updated the URL to the new repo

Reply:
Since iconUrl is deprecated, can you replace with an icon element pointing to vsix/cppwinrt.ico, and add that file to the nuspec?